charts/argo-cd/Chart.yaml

```diff
@@ -27,3 +27,4 @@ annotations:
     - "[Deprecated]: Configuration option configs.gpgKeys"
     - "[Deprecated]: Configuration option configs.gpgKeysAnnotations"
     - "[Fixed]: Documentation for declarative setup"
```

Comment:
Could you please remove old changes as it gets merged when published?

Reply:
Done